bindings/python/README.txt

```diff
@@ -1,13 +1,17 @@
-NOTE: This package includes prebuilt Windows core of Capstone, so no external
-Capstone library is needed.
+To install capstone, you should run `pip install capstone`.
 
-On Windows, download this PyPi package and run "python setup.py install" from
-"Command Prompt" is all you need to install it.
+If you would like to build capstone with just the source distribution, without
+pip, just run `python setup.py install` in the folder with setup.py in it.
 
-If you want to compile everything from source code instead, use the
-PyPi package "capstone" instead.
+In order to use this source distribution, you will need an environment that can
+compile C code. On linux, this is usually easy, but on windows, this involves
+installing Visual Studio and using the "Developer Command Prompt" to perform the
+installation. See BUILDING.txt for more information.
 
-    https://pypi.python.org/pypi/capstone
+If you don't want to build your own copy of capstone, you can use a precompiled
+binary distribution frm PyPI. Saying `pip install capstone` should automatically
```

Comment:
typo here on "frm". also, names should be capitalized, such as "Linux", "Windows", "Capstone", etc